### ISSUE TITLE
Import does not work

### ISSUE THREAD

**Issue by Erebus999** [nextcloud/bookmarks#816]
Export all bookmarks ..ok
Import delete alle bookmarks an the list ist empty.
Import does not work on Nextcloud 16.0.5 and 17 RC2

**Comment by dave-kennedy** [comment]
I'm having the same issue.

**To Reproduce**
1. Go to Bookmarks
2. Click on Settings
3. Click on Import
4. Select a bookmarks file to import

**Expected Behavior**
Import succeeds.

**Actual Behavior**
Nothing happens.

**Screenshots**
![Screenshot from 2019-10-06 00-36-00](https://user-images.githubusercontent.com/13735532/66265297-5b73af00-e7d1-11e9-8f11-58ef5c2e1b2d.png)

![Screenshot from 2019-10-06 00-54-24](https://user-images.githubusercontent.com/13735532/66265466-2ae14480-e7d4-11e9-8a2d-cf5a2aba2db5.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 19.04
 - Browser: Firefox 69.0.1

**Server (please complete the following information):**
 - OS: Raspbian Buster Lite 4.19.66
 - HTTP server: Apache 2.4.38
 - Database: MariaDB 10.3.17
 - PHP version: 7.3.9
 - Nextcloud version: 17.0.0
 - Bookmarks app version: 2.1.1
 - Activated Nextcloud Apps:
```
Enabled:
  - accessibility: 1.3.0
  - activity: 2.10.1
  - bookmarks: 2.1.1
  - bruteforcesettings: 1.4.0
  - calendar: 1.7.1
  - cloud_federation_api: 1.0.0
  - comments: 1.7.0
  - contacts: 3.1.6
  - dav: 1.13.0
  - federatedfilesharing: 1.7.0
  - federation: 1.7.0
  - files: 1.12.0
  - files_pdfviewer: 1.6.0
  - files_rightclick: 0.15.1
  - files_sharing: 1.9.0
  - files_trashbin: 1.7.0
  - files_versions: 1.10.0
  - files_videoplayer: 1.6.0
  - firstrunwizard: 2.6.0
  - gallery: 18.4.0
  - logreader: 2.2.0
  - lookup_server_connector: 1.5.0
  - nextcloud_announcements: 1.6.0
  - notes: 3.0.3
  - notifications: 2.5.0
  - oauth2: 1.5.0
  - password_policy: 1.7.0
  - privacy: 1.1.0
  - provisioning_api: 1.7.0
  - recommendations: 0.5.0
  - serverinfo: 1.7.0
  - sharebymail: 1.7.0
  - support: 1.0.1
  - survey_client: 1.5.0
  - systemtags: 1.7.0
  - text: 1.1.0
  - theming: 1.8.0
  - twofactor_backupcodes: 1.6.0
  - updatenotification: 1.7.0
  - viewer: 1.1.0
  - workflowengine: 1.7.0
Disabled:
  - admin_audit
  - encryption
  - files_external
  - user_ldap
```
 - Nextcloud configuration:
```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/nextcloud\/",
        "htaccess.RewriteBase": "\/",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "nextcloud"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "17.0.0.9",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "maintenance": false
    }
}
```
 - Nextcloud external user backend: none

**Web server error log**
```
No errors are present in the server log.
```

**Nextcloud log (nextcloud/data/nextcloud.log)**
```
Nada.
```

**Browser log**
```
No errors appear in the JavaScript console. A DELETE request is made to "/bookmark", followed by a GET to "/folder", as captured in the screenshot above.

**Comment by dave-kennedy** [comment]
Derp, nevermind. I see now that it's been fixed in 2.2.0.
